addons/delivery/static/src/js/location_selector/map_container/map_container.scss

```diff
@@ -0,0 +1,17 @@
+.o_location_selector_skeleton_title {
+    height: $h5-font-size * $headings-line-height;
+}
+
+.o_location_selector_skeleton_content {
+    height: $small-font-size;
+    margin-bottom: ($line-height-base * $small-font-size) - $small-font-size;
+}
+
+.o_location_selector_skeleton_day {
+    width: 9ch;
+}
+
+.o_location_selector_skeleton_btn {
+    height: $btn-font-size * $btn-line-height + $btn-padding-y * 2;
```

Comment:
Hello @Brieuc-brd 

I've got an issue on runbot: `Incompatible units: 'px' and 'rem'.` Do you have any insight about how to fix it ?